cli/src/commands/start.rs

```diff
@@ -262,7 +262,7 @@ impl Start {
         // If the display is not enabled, render the welcome message.
         if self.nodisplay {
             // Print the Aleo address.
-            println!("🪪 Your Aleo address is {}.\n", account.address().to_string().bold());
+            println!("🪪  Your Aleo address is {}.\n", account.address().to_string().bold());
```

Comment:
Why this extra space? Looks better? Or a mistake?

Reply:
Yeah, the space doesn't render on my machine, maybe it's fine for you though? If so, I'll revert.

Reply:
Indeed, the emoji seems to wide, the Y is over it. 👍